src/tribol/physics/CommonPlane.cpp

```diff
@@ -157,302 +156,301 @@ real ComputeGapRatePressure( ContactPlaneManager& cpMgr,
+        RealT totalPressure = 0.;
+        plane.m_pressure = gap * penalty_stiff_per_area; // kinematic contribution
+        switch(pen_enfrc_options.constraint_type)
+        {
+          case KINEMATIC_AND_RATE:
+          {
+              // kinematic contribution
+              totalPressure += plane.m_pressure;
+              // add gap-rate contribution
+              totalPressure += 
+                ComputeGapRatePressure( plane, mesh1, mesh2, penalty_stiff_per_area,
+                                        pen_enfrc_options.rate_calculation );
+              break;
+          }
+          case KINEMATIC:
+              // kinematic gap pressure contribution  only
+              totalPressure += plane.m_pressure;
+              break;
+          default:
+              // no-op
+              break;
+        } // end switch on registered penalty enforcement option
+
+        // debug prints. Comment out for now, but keep for future common plane 
+        // debugging
+  //         SLIC_DEBUG("gap: " << gap);
+  //         SLIC_DEBUG("area: " << A);
+  //         SLIC_DEBUG("penalty stiffness: " << penalty_stiff_per_area);
+  //         SLIC_DEBUG("pressure: " << cpManager.m_pressure[ cpID ]);
+
+        ///////////////////////////////////////////
+        // create surface contact element struct //
+        ///////////////////////////////////////////
+
+        // construct array of nodal coordinates
+        constexpr int max_dim = 3;
+        constexpr int max_nodes_per_face = 4;
+        constexpr int max_nodes_per_overlap = 8;
+        RealT xf1[ max_dim * max_nodes_per_face ];
+        RealT xf2[ max_dim * max_nodes_per_face ];
+        RealT xVert[ max_dim * max_nodes_per_overlap ];  
+        int dim = cs_view.spatialDimension();
+        int num_nodes_per_face = mesh1.numberOfNodesPerElement();
+        initRealArray( xf1, dim * num_nodes_per_face, 0. );
+        initRealArray( xf2, dim * num_nodes_per_face, 0. );
+        auto xVert_size = 4;
+        auto numPolyVert = 2;
+        if (dim == 3)
+        {
+          auto& cp3 = static_cast<ContactPlane3D&>(plane);
+          numPolyVert = cp3.m_numPolyVert;
+          xVert_size = 3 * numPolyVert;
+        }
+        initRealArray( xVert, xVert_size, 0. );
 
-      //////////////////////////////////////////////////////
-      // loop over nodes and compute contact nodal forces //
-      //////////////////////////////////////////////////////
-      for( IndexType a=0 ; a < numNodesPerFace ; ++a )
-      {
+  //      // get projected face coordinates
+  //      cpManager.getProjectedFaceCoords( cpID, 0, &xf1[0] ); // face 0 = first face
+  //      cpManager.getProjectedFaceCoords( cpID, 1, &xf2[0] ); // face 1 = second face
 
-        IndexType node0 = nodalConnectivity1[ index1*numNodesPerFace + a ];
-        IndexType node1 = nodalConnectivity2[ index2*numNodesPerFace + a ];
+        // get current configuration, physical coordinates of each face
```

Comment:
Look at formatting. Looks like indentation is based on a deleted for loop above.

Reply:
Looks OK to me; verify on the source file.